src/lib/checkbox/checkbox.ts

```diff
@@ -217,15 +217,15 @@ export class CheckboxComponent
   /**
    * Gets/sets whether the checkbox is checked.
    * @default false
-   * @attribute
+   * @attribute checked
```

Comment:
Just an FYI, for properties and attributes that are the same name you don't need to specify the attribute name here since it will use the property name by default.